Modules/TRD/include/TRD/Noise.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   Noise.h
+/// \author My Name
+///
+
+#ifndef QC_MODULE_TRD_TRDNOISE_H
+#define QC_MODULE_TRD_TRDNOISE_H
+
+#include "QualityControl/TaskInterface.h"
+
+class TH1F;
+
+using namespace o2::quality_control::core;
+
+namespace o2::quality_control_modules::trd
+{
+
+/// \brief Example Quality Control DPL Task
```

Comment:
and update the description ?

Reply:
I just removed Noise.* and updated the descriptions in the other files.